### DISCUSSION TITLE
Comments under pictures

### DISCUSSION THREAD

**Discussion by dontlookherepls** [marp-team/marp#239] [Q&A] [answered]
Hi ! How can i add text under picture without html ? 

(yeah ik it is problably a dumb question but i can't find any source)

**Comment by yhatt** [comment, marked as answer]
I caught as that it was perhaps meaning image caption.

As described in the below post, using `<figcaption>` HTML tag is exactly correct way from a view of both semantics of Markdown and HTML. We recommend to follow the standard approach due to making less compatibility as the Markdown document.
👉 [Captions in Markdown | theSynAck](https://thesynack.com/posts/markdown-captions/)

If you really want the caption comment without html, [you have to style a text placed after image treated as the caption.](https://thesynack.com/posts/markdown-captions/#targeting-captions-with-css) If you're using Marp CLI, you also can try to use a markdown-it plugin like [markdown-it-implicit-figures](https://github.com/arve0/markdown-it-implicit-figures) and [markdown-it-image-figures](https://github.com/Antonio-Laguna/markdown-it-image-figures). They can generate `<figcaption>` from the title for Markdown image.

In any case, you have to define the style in custom theme CSS or [inline style](https://marpit.marp.app/theme-css?id=tweak-style-through-markdown) to get better looking caption.

**Comment by hamzamohdzubair** [comment]
if i understand correctly alt text in bg images gets converted to figcaption (https://github.com/marp-team/marpit/pull/371) but alt text of non-bg images does not get converted to figcaption? Is this understanding correct? I am unable to find much in the documentation, other than these two issues, one of which (this one) being 4 years old. 

Just want to confirm if using markdown-it plugins is still the only way to solve this problem. Thanks in advance for your efforts, for marp and the marp family of software.

**Reply by yhatt** [reply]
Yes. For regular images that are not background images (e.g. `![alternative text](image.jpg)`), Markdown will convert it to `<img alt="alternative text" />` instead of `<figure>` and `<figcaption>`.

It's also a spec of CommonMark, and we should follow it if the image was not marked as a background image by `bg` keyword.
https://spec.commonmark.org/0.31.2/#images

> Just want to confirm if using markdown-it plugins is still the only way to solve this problem.

In my opinion, there is no need to use Marp / markdown-it plugin just to put the caption below an image. It's just enough to write the following Markdown:

```markdown
![Marp logo](https://marp.app/assets/marp.svg)

<figcaption>Marp: Markdown Presentation Ecosystem</figcaption>
```

As described in [the mentioned article](https://thesynack.com/posts/markdown-captions/), the `<figcaption>` element is semantically appropriate for image captions. Keeping semantics of elements in the slide clear is already important for accessibility. Even if the HTML tag used in Markdown, [it's still the valid Markdown](https://spec.commonmark.org/0.31.2/#html-blocks).

But if your slide does not worry about accessibility and does not want to use HTML tag anyway, this example can be much simpler:

```markdown
![Marp logo](https://marp.app/assets/marp.svg)

Marp: Markdown Presentation Ecosystem
```

Yes, just put the caption text below the image. :)
